packages/nuejs.org/docs/css-best-practices.md

```diff
@@ -23,7 +23,7 @@ These are all described in this document. Once you harness the power of CSS you
 
 
 ## Respect constraints { #constraints }
-Limit yourself to as few fonts, font weights, colors, variables, elements, class names, and components as possible. This is beneficial for several reasons:
+Limit yourself to as few fonts, font weights, colors, variables, elements, class names and components as possible. This is beneficial for several reasons:
```

Comment:
My current standard is to use the "oxford comma", or is that a bad practice?

Reply:
This is a minefield or course but you usually put a comma before and when it’s connecting two independent clauses. It’s almost always optional to put a comma before and in a list. I would suggest to adhere to that rule.